app/components/polls/statements/CardButton.module.css

```diff
@@ -1,35 +1,48 @@
-.agree[data-active="true"], .agree:hover {
-  background: linear-gradient(
+.agree {
+  --button-background: linear-gradient(
       178.91deg,
       rgba(255, 255, 255, 0) 59.39%,
       rgba(255, 255, 255, 0.39) 99.07%
-    ),
-    linear-gradient(180deg, #7857ff 0%, #483499 100%);
-  box-shadow:
-    0 0 0 2px theme("colors.zinc.800/10%"),
-    0px 0px 20px 0px rgba(120, 87, 255, 0.9);
+  ),
+  linear-gradient(180deg, #7857ff 0%, #483499 100%);
+  --button-shadow: 0 0 0 2px theme("colors.zinc.800/10%"),
+  0px 0px 20px 0px rgba(120, 87, 255, 0.9)
 }
 
-.disagree[data-active="true"], .disagree:hover {
-  background: linear-gradient(
+.disagree {
+  --button-background: linear-gradient(
       178.91deg,
       rgba(255, 255, 255, 0) 59.39%,
       rgba(255, 255, 255, 0.39) 99.07%
-    ),
-    linear-gradient(180deg, #ff4975 0%, #f83160 100%);
-  box-shadow:
-    0 0 0 2px theme("colors.zinc.800/10%"),
-    0px 0px 20px 0px rgba(255, 73, 117, 0.9);
+  ),
+  linear-gradient(180deg, #ff4975 0%, #f83160 100%);
+  --button-shadow: 0 0 0 2px theme("colors.zinc.800/10%"),
+  0px 0px 20px 0px rgba(255, 73, 117, 0.9);
 }
 
-.skip[data-active="true"], .skip:hover {
-  background: linear-gradient(
+.skip {
+  --button-background: linear-gradient(
       178.91deg,
       rgba(255, 255, 255, 0) 59.39%,
       rgba(255, 255, 255, 0.1) 99.07%
-    ),
-    linear-gradient(180deg, #332c2d 0%, #998589 100%);
-  box-shadow:
-    0 0 0 2px theme("colors.zinc.800/10%"),
-    0px 0px 20px 0px rgba(102, 86, 88, 1);
+  ),
+  linear-gradient(180deg, #332c2d 0%, #998589 100%);
+  --button-shadow: 0 0 0 2px theme("colors.zinc.800/10%"),
+  0px 0px 20px 0px rgba(102, 86, 88, 1);
+}
+
+/* Apply hover effect only on non-touch devices */
+/* Source: https://stackoverflow.com/a/64553121 */
```

Comment:
Love that you added the source in a comment! 🙏🏻

Reply:
Thanks! Glad it was helpful.